db/warehouse/migrate/20230407164611_add_external_referrals.rb

```diff
@@ -0,0 +1,52 @@
+class AddExternalReferrals < ActiveRecord::Migration[6.1]
+  def change
+    create_table :unit_types do |t|
+      t.timestamps
+      t.string :description
+      t.integer :bed_type # HUD bed type
+      t.integer :unit_size
+      # checking... this would be external_id_id. Is this right?
```

Comment:
I think we settled on `external_id.value` for the storage of the external identifier, but yeah, if it's referencing the active record model, then I think it would end up being `external_id_id`.  That begs the question of how tightly tied these should be, maybe it should just be `external_id` which would match `external_ids.value` but require both be obtained through external systems.  (As in you could have an `external_id` here that didn't necessarily exist yet in `external_ids`

Reply:
removed the external id reference per thread above https://github.com/greenriver/hmis-warehouse/pull/2955#discussion_r1161658441